etc/kayobe/kolla/config/monasca/agent_plugins/ntp.yaml

```diff
@@ -0,0 +1,6 @@
+{% raw %}
+init_config: null
+instances:
+- built_by: Ntp
+  host: "{{ external_ntp_servers | first }}"
```

Comment:
Are we setting this variable anywhere?

Reply:
There's nothing set in `etc/kayobe/ntp.yml` at the moment, this just relies on the Kolla-Ansible default which happens to be the same as the default generated by monasca-setup (which we don't use at the moment).

Reply:
Should kayobe pass its NTP config through to external_ntp_servers? Currently it doesn't.

Reply:
Yeah, it's probably worth a patch now that Monasca is using it, will add it to my TODO list. Previously `external_ntp_servers` was exclusively used for chronyd in Kolla Ansible.